src/ExportGenerators.mss

```diff
@@ -1422,7 +1422,12 @@ function GenerateStaffGroups (score, barnum) {
 function GenerateTuplet(tupletObj) {
     //$module(ExportGenerators.mss)
     tuplet = libmei.Tuplet();
+    dur = tupletObj.PlayedDuration;
+    meidur = ConvertDuration(dur);
```

Comment:
`PlayedDuration` should not be used here because it can have funny values like `341` that don't make sense for `ConvertDuration()`.

I'd rather not add `@dur` unless we do it properly (I'm also ). Any argument for having it anyway?

Reply:
That's odd. But maybe I simply missunderstood the ManuScript Manual here.